### PR TITLE
getUsername overridden

### PR DESCRIPTION
**Pull request by hedgehog1833**
The UserEntity's getUsername() now returns the email to achieve having a distinct value in the username-column in the persistent logins table.
I had to add the username as a model in the HomeController as thymeleaf took it by some magic using the getUsername() method, so we could not use this anymore as the email is returned there now.
But all in all it looks better than the custom remember me service I guess.

### CODE REVIEW

**Comment by hedgehog1833** [comment]
[Custom PersistentRememberMeService](https://trello.com/c/6LmVvD1u/338-custom-persistentremembermeservice)

**Comment by hedgehog1833** [comment]
If we go this way we should truncate the persistent_logins table, either manually or via flyway-script!

File: persistence/src/main/java/rocks/metaldetector/persistence/domain/user/UserEntity.java [hunk 1]
```diff
@@ -95,6 +95,14 @@ public void setEmail(String newEmail) {
     email = newEmail == null ? "" : newEmail;
   }
 
+  public String getUsername() {
+    return email;
+  }
```
Comment (DanielW1987) [inline persistence/src/main/java/rocks/metaldetector/persistence/domain/user/UserEntity.java:100]: I don't like comments in code but maybe here is a little bit explanation meaningful

**Review by DanielW1987 — COMMENTED** [review]
I think we should clear the table by hand.

**Comment by DanielW1987** [comment]
...looks really good to me 👍

**Review by DanielW1987 — APPROVED** [review]

**Pull request merged**